tests/x11/thunderbird/thunderbird_pop.pm

```diff
@@ -22,6 +22,7 @@ use strict;
 use testapi;
 use utils;
 use base "thunderbird_common";
+use x11utils qw(ensure_unlocked_desktop turn_off_gnome_screensaver turn_off_gnome_suspend);
```

Comment:
you are not introducing any new uses of these functions, why import?

Reply:
Was used in other moment, removed. My lasts commits was not very happy, really.